GW2EIEvtcParser/EIData/ProfHelpers/ProfHelper.cs

```diff
@@ -150,9 +150,11 @@ internal static class ProfHelper
             .UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
         new EffectCastFinder(RelicOfSorrowBuff, EffectGUIDs.RelicOfSorrow3)
             .UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
-        // If this collides with something else we could check via buff remove with the condition dst == src
         new EffectCastFinder(RelicOfTheStormsingerChain, EffectGUIDs.RelicOfTheStormsinger)
-            .UsingSecondaryEffectChecker(EffectGUIDs.RelicOfTheStormsinger) // Effect triggers at least twice
+            .UsingChecker((effectEvent, combatData, agentData, skillData) => 
+            {
+                return combatData.GetBuffData(RelicOfTheStormsingerBuff).Where(x => x is BuffRemoveAllEvent && Math.Abs(x.Time - effectEvent.Time) < ServerDelayConstant).FirstOrDefault() != null;
```

Comment:
combatData.HasLostBuff API can be used here instead.
Perhaps we should leave the double effect check also to be safe?

Reply:
The incorrect trigger on the htcm log was caused by 4 effects triggering within roughly 2 seconds, so i've removed it entirely